mne/viz/topomap.py

```diff
@@ -237,11 +236,11 @@ def _check_outlines(pos, outlines, head_scale=0.85):
         ear_y = np.array([.0555, .0775, .0783, .0746, .0555, -.0055, -.0932,
                           -.1313, -.1384, -.1199])
         x, y = pos[:, :2].T
-        x_range = np.abs(x.max() - x.min())
-        y_range = np.abs(y.max() - y.min())
+        x_range = max(x.max(), -x.min()) * 2.
+        y_range = y.max() - y.min()
 
         # shift and scale the electrode positions
-        pos[:, 0] = head_scale * ((pos[:, 0] - x.min()) / x_range - 0.5)
+        pos[:, 0] = head_scale * ((pos[:, 0]) / x_range)
         pos[:, 1] = head_scale * ((pos[:, 1] - y.min()) / y_range - 0.5)
 
```

Comment:
this is the part of the code that forces the center to be @ x=0 (#2113) instead of in the middle of the (subset of the) sensor layout